librz/io/p/io_rzpipe.c

```diff
@@ -39,13 +38,8 @@ static int __write(RzIO *io, RzIODesc *fd, const ut8 *buf, int count) {
 		eprintf("rzpipe_write: error\n");
 		return -1;
 	}
-	res = rzpipe_read(RZP(fd));
```

Comment:
sure about this?? Maybe `res` is dead, but the call could have other effects, right?